### DISCUSSION TITLE
group finding by target

### DISCUSSION THREAD

**Discussion by OlderMutt** [Syslifters/sysreptor#337] [Q&A]
Hi all,
i need to group the findings not by title.
i have a penetration test with 30 targets and i would like to group the finding for IPs/hostname.
is there a way to modify the findings in this way ?
thanks all in advance

**Comment by MWedl** [comment]
Hi,
it is possible to group findings into separate chapters in the PDF report by grouping them via VueJS template language during PDF rendering. See https://github.com/Syslifters/sysreptor/issues/90 and https://github.com/Syslifters/sysreptor/discussions/4 for examples.

The finding sidebar currently only supports a flat list. However, you can customize the finding ordering (see https://github.com/Syslifters/sysreptor/issues/69) to sort findings by a hostname field (e.g. combobox with hostname/IP), then by CVSS. By ordering via a hostname field first, findings of the same hostname appear next to each other in the list. There are no group headlines, though.

**Comment by aronmolnar** [comment]
This is now possible. See: https://docs.sysreptor.com/designer/findings/#finding-groups
